src/suspense/component.tsx

```diff
@@ -23,43 +60,46 @@ export class LazySuspense extends Component<
     setFallback: (fallback: Fallback) => {
       if (this.hydrationFallback === fallback) return;
       this.hydrationFallback = fallback;
-      this.useSibling = Boolean(fallback);
       // Schedule an update so we force switch from the sibling tree
       // back to the suspense boundary
-      this.forceUpdate();
+      if (this.mounted) this.forceUpdate();
```

Comment:
Wait mount before forcing update, otherwise React complains. If `setFallback` is called before, we rely on rendering order: Suspense is rendered before the dynamic fallback sibling so it's enough to get the SSR content properly rendered anyway.